kbs/src/api/src/attestation/mod.rs

```diff
@@ -14,6 +14,11 @@ use intel_trust_authority::*;
 use kbs_types::{Challenge, Tee};
 use rand::{thread_rng, Rng};
 
+#[cfg(any(feature = "coco-as-builtin", feature = "coco-as-grpc"))]
```

Comment:
Should this just be `#[cfg(not(feature = "intel-trust-authority-as"))]`?

Reply:
that works also. the proposed `cfg` was a copy-paste from the other coco-as* configs in this file.